run-tester.ps1

```diff
@@ -50,10 +50,7 @@ $atprocess = Start-Job -Init ([ScriptBlock]::Create("Set-Location '$pwd\nvda-at-
 Write-Output "Waiting for localhost:3031 to start from at-driver"
 Wait-For-HTTP-Response -RequestURL http://localhost:3031
 
-Write-Output "Starting chromedriver"
-$chromeprocess = Start-Job -Init ([ScriptBlock]::Create("Set-Location '$pwd'")) -ScriptBlock { chromedriver --port=4444 --log-level=INFO *>&1 >$using:loglocation\chromedriver.log }
-Write-Output "Waiting for localhost:4444 to start from chromedriver"
-Wait-For-HTTP-Response -RequestURL http://localhost:4444/
+. .\start-$env:BROWSER.ps1
```

Comment:
This is concise, but I've grown weary of generating file names in scripts. Reason being: file names are a useful handle for tracing execution flow, and I've been frustrated in situations where I couldn't use `grep` to answer questions like, "where is this thing used?" `start-firefox.ps1` and `start-chrome.ps1` are misnomers, anyway, since they don't launch web browsers.

Can we use the variable to decide between literal values like `start-chromedriver.ps1` and `start-geckodriver.ps1`? Or could we implement this as a single script named `start-webdriver-server.ps1` and pass it the browser name as an argument?

Reply:
did this with just a switch statement in the single file